tests/zfs-tests/tests/functional/rsend/rsend_001_pos.ksh

```diff
@@ -64,6 +64,9 @@ log_must cleanup_pool $POOL2
 # Verify all the filesystem and sub-fs can be backup and restored.
 #
 log_must eval "zfs send -R $POOL/$FS@final > $BACKDIR/fs-final-R"
+if is_freebsd; then
+	sleep 10
+fi
```

Comment:
No objection, but I'm curious why a sleep is needed here on FreeBSD.  I don't think it should be required between a send/recv unless there's a problem.  If it is, a comment explaining why would be good.

Reply:
I've removed the FreeBSD sleeps from the rsend tests.